daffodil-runtime2/src/main/resources/org/apache/daffodil/runtime2/c/libruntime/parsers.c

```diff
@@ -507,14 +507,21 @@ parse_le_uint8(uint8_t *number, size_t num_bits, PState *pstate)
     *number = (uint8_t)integer;
 }
 
-// Parse fill bits until end bitPos0b is reached
+// Parse fill bits up to alignmentInBits or end_bitPos0b
+
+void
+parse_align(size_t alignmentInBits, PState *pstate)
+{
+    size_t end_bitPos0b = (pstate->bitPos0b + alignmentInBits - 1) / alignmentInBits * alignmentInBits;
```

Comment:
Please add the implied parentheses here. At first I though you were dividing by alignmentInBits squared. 

The fact that this works to compute what the ending pos0b should be is not actually obvious.  I expected to see modular division operator ( x % y to get the remainder) here to figure out the amount of fill.